docs/ide/reference/options-text-editor-csharp-advanced.md

```diff
@@ -30,56 +30,64 @@ Use the **Advanced** options page to modify the settings for editor formatting,
 
 - Place 'System' directives first when sorting usings
 
-   When selected whenever you select “Sort Usings” or “Remove and Sort”, Visual Studio will sort the usings statements and ensure that all the system directives are ordered first.
-
-Pre Sort
-
-    using AutoMapper;
-    using FluentValidation;
-    using System.Collections.Generic;
-    using System.Linq;
-    using Newtonsoft.Json;
-    using System;
-
-After Sort
-
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using AutoMapper;
-    using FluentValidation;
-    using Newtonsoft.Json;
-
+   When selected, when you select **Remove and Sort Usings**, Visual Studio sorts the `using` directives and places the 'System' namespaces at the top of the list.
+
+   Before sorting:
+
+   ```csharp
+   using AutoMapper;
+   using FluentValidation;
+   using System.Collections.Generic;
+   using System.Linq;
+   using Newtonsoft.Json;
+   using System;
+   ```
+   
+   After sorting:
+
+   ```csharp
+   using System;
+   using System.Collections.Generic;
+   using System.Linq;
+   using AutoMapper;
+   using FluentValidation;
+   using Newtonsoft.Json;
+   ```
+   
 - Separate using directive groups
 
-   When selected, whenever you select “Sort Usings” or “Remove and Sort”, Visual Studio will sort the usings by directive group.
-
-Pre Sort
-
-    using AutoMapper;
-    using FluentValidation;
-    using System.Collections.Generic;
-    using System.Linq;
-    using Newtonsoft.Json;
-    using System;
-
-After Sort
-
-    using AutoMapper;
-    
-    using FluentValidation;
-    
-    using Newtonsoft.Json;
-    
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-
+   When selected, when you select **Remove and Sort Usings**, Visual Studio separates `using` directives by inserting an empty line between groups of directives.
```

Comment:
Same here.

Reword the phrase "When selected, when you select..." to "When selected, the **Remove and Sort Usings** option in the right-click menu separates 'using' directives by inserting an empty line between groups of directives."